vmm/task/src/main.rs

```diff
@@ -166,6 +184,22 @@ async fn main() {
     handle_signals(signals).await;
 }
 
+// Do some initialization before everything starts.
+// Such as setting envs, preparing cgroup mounts, setting kernel paras.
+async fn early_init_call() -> Result<()> {
+    // Set environment variables from ENVS vector(ordered).
+    for (k, v) in ENVS.iter() {
+        std::env::set_var(k, v);
+    }
+    // Set kernel parameters from KERNEL_PARAS vector(ordered).
+    for (_path, _v) in KERNEL_PARAS.iter() {
+        // TODO: write kernel paras
```

Comment:
kernel params may got from cmdline params, The static params may be compiled into the kernel.

Reply:
I'd like to remove code of setting kernel paras as we dont have any paras for now.